Code/IO/include/sitkImageSeriesReader.h

```diff
@@ -119,7 +119,8 @@ namespace itk {
        * \param directory  The directory that contains the DICOM data set
        * \sa itk::GDCMSeriesFileNames
        **/
-      static std::vector<std::string> GetGDCMSeriesIDs( const std::string &directory );
+      static std::vector<std::string> GetGDCMSeriesIDs( const std::string &directory,
```

Comment:
Should we document in the function documentation that the value of `useSeriesDetails` should match when the GetGDCMSeriesIDs/FileNames is used?

Reply:
Good catch. Documentation updated.